staging/src/k8s.io/api/extensions/v1beta1/doc.go

```diff
@@ -17,5 +17,6 @@ limitations under the License.
 // +k8s:deepcopy-gen=package
 // +k8s:protobuf-gen=package
 // +k8s:openapi-gen=true
+// +k8s:status-gen=true
```

Comment:
I think most people will expect this to do something related to a status subresource... not sure what would be clearer... lifecycle or something?

Reply:
> I think most people will expect this to do something related to a status subresource... not sure what would be clearer... lifecycle or something?

`beta-lifecycle-gen`?